owncloudApp/src/main/java/com/owncloud/android/ui/activity/ManageSpaceActivity.java

```diff
@@ -108,10 +107,10 @@ protected Boolean doInBackground(Void... params) {
 
 
             final String passcodeString = appPrefs.getString(PREFERENCE_PASSCODE, null);
-            final String[] passCodeDigits = new String[PassCodeActivity.numberOfPassInputs];
+            final String[] passCodeDigits = new String[getBaseContext().getResources().getInteger(R.integer.passcode_digits)];
```

Comment:
You could create a new variable, for example `numberOfPasscodeInputs`, and initialize it to `getBaseContext().getResources().getInteger(R.integer.passcode_digits)` so you don't need to call the `getResources` several times in the same method.